nautobot_chatops_ipfabric/worker.py

```diff
@@ -198,23 +266,23 @@ def get_inventory(dispatcher, filter_key=None, filter_value=None):
                 "Device Inventory",
                 ipfabric_logo(dispatcher),
             ),
-            dispatcher.markdown_block(f"{ipfabric_api.host_url}/inventory/devices"),
+            dispatcher.markdown_block(f"{ipfabric_api.client.base_url}inventory/devices"),
```

Comment:
not necessary to change in this PR, but it looks like you could use the path constants from above

Reply:
will leave as they are for now. Need to check on the v4 UI whenever we get access. REST URL has `tables/..` as prefix.